apps/studio/components/layouts/LogsLayout/LogsLayout.tsx

```diff
@@ -15,36 +15,35 @@ interface LogsLayoutProps {
 }
 
 const LogsLayout = ({ title, children }: PropsWithChildren<LogsLayoutProps>) => {
-  const { isLoading, can: canUseLogsExplorer } = useAsyncCheckProjectPermissions(
-    PermissionAction.ANALYTICS_READ,
-    'logflare'
-  )
+  const {
+    isLoading,
+    isSuccess,
+    can: canUseLogsExplorer,
+  } = useAsyncCheckProjectPermissions(PermissionAction.ANALYTICS_READ, 'logflare')
 
   const router = useRouter()
   const [_, setLastLogsPage] = useLocalStorageQuery(
     LOCAL_STORAGE_KEYS.LAST_VISITED_LOGS_PAGE,
-    router.pathname.split('/').pop()
```

Comment:
reckon this should be the same behaviour as logs/index.tsx - default to /explorer